SCClassLibrary/Common/Control/Volume.sc

```diff
@@ -4,37 +4,96 @@ Volume {
 
 	var <volume = 0.0, <lag = 0.1, <isMuted = false;
 
-	var <ampSynth, <numOutputChannels, defName, updateFunc, initFunc;
+	var <ampSynth, <numOutputChannels, <defName, updateFunc, initFunc;
 	var <>window;
+	var <group, <ampSynthRunning = false, <listener;
 
 	*new { | server, startBus = 0, numChannels, min = -90, max = 6, persist = false |
 		^super.newCopyArgs(server ?? { Server.default }, startBus, numChannels, min, max, persist).init;
 	}
 
 	init {
+		// hardcode group nodeID to 2, so multiple clients can use it
+		group = Group.basicNew(server, 2);
+		defName = (\volumeAmpControl ++ this.numChannels).asSymbol;
+		// hardcode synth nodeID to 3, so multiple clients can use it
+		ampSynth = Synth.basicNew(defName, server, 3);
+		ampSynth.group = group;
+
 		// execute immediately if we're already past server tree functions
 		if(server.serverRunning) {
 			this.sendSynthDef;
 			this.updateSynth;
 		};
 
 		initFunc = {
-			ampSynth = nil;
+			ampSynthRunning = false;
```

Comment:
couldn't this be done with `Group(...).register`?

Reply:
I guess yes. What would the advantages of registering be? 

As it is, we already track whether the node is alive or not by its volume status messages. 
The only case I see where that misses a change is when ```s.volume.ampSynth.free``` is called,
because then the ampSynth does not send a last triggered update with gate 0.

Reply:
I though that it may be easier to read and clearer expression of intent.

Reply:
yes, you are right. with onFree, this catches irregular ends to ampSynth as well. thanks!